addons/trenches/functions/fnc_getSurfaceTexturePathNative.sqf

```diff
@@ -0,0 +1,29 @@
+#include "script_component.hpp"
+/*
+ * Author: chris579
+ * Gets the path to the ground texture under the object
+ * Native version using intercept cba 'surfaceTexture'
```

Comment:
```suggestion
 * Native version using Intercept CBA 'surfaceTexture',
```

Reply:
if you use comma here, the `Is` on next line should probably be lowercase?